projects/demo/tsconfig.app.json

```diff
@@ -4,7 +4,13 @@
     "include": ["src/**/*.ts"],
     "angularCompilerOptions": {
         "strictMetadataEmit": false,
-        "compilationMode": "full"
+        "compilationMode": "full",
+        "extendedDiagnostics": {
+            "checks": {
+                "unusedStandaloneImports": "warning"
+            },
+            "defaultCategory": "error"
+        }
```

Comment:
what if?

```
"extendedDiagnostics": {
    "defaultCategory": "warning"
}
```

Reply:
My mistake, it works, I've checked it now